README.md

```diff
@@ -29,3 +29,14 @@ iTunes feed that uses m4a with a fallback to mp3 if there is no m4a for that epi
     /podcast_feed/talk/itunes/rss.xml
 
 Note: You can also add a feed_size parameter and a page parameter to the URL ("?feed_size=4&page=3")
+
+
+# Stork search
```

Comment:
@makefu please link to how one would install stork

Reply:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/misc/stork/default.nix :
```
NIX_PATH=nixpkgs=/path/to/unstable nix-shell -p stork
```
es gibt auch eine nix action: https://github.com/marketplace/actions/install-nix

Reply:
We can also build the package with
```
cargo build
cargo run
```

Reply:
zusätzlich gibt es noch die github action vom stork search: https://github.com/jameslittle230/stork/blob/master/.github/workflows/create-release-after-tag.yml